### ISSUE TITLE
Issue in start_training

### ISSUE THREAD

**Issue by Sunil13kush** [0dust/ResumeFilter#13]
python3 start_training.py
2024-04-24 18:27:39.079680: I tensorflow/core/util/port.cc:113] oneDNN custom operations are on. You may see slightly different numerical results due to floating-point round-off errors from different computation orders. To turn them off, set the environment variable `TF_ENABLE_ONEDNN_OPTS=0`.
2024-04-24 18:27:39.079914: I external/local_tsl/tsl/cuda/cudart_stub.cc:32] Could not find cuda drivers on your machine, GPU will not be used.
2024-04-24 18:27:39.082191: I external/local_tsl/tsl/cuda/cudart_stub.cc:32] Could not find cuda drivers on your machine, GPU will not be used.
2024-04-24 18:27:39.110750: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-04-24 18:27:39.578656: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
###################### Training for line label ###########################
ready to fit training data..
shape of x_train,x_test,y_train,y_test>> (51, 11) (22, 11) (51, 5) (22, 5)
loading embeddings...
/home/lenovo3/resume-parsing/ResumeFilter-master/data/glove.6B/glove.6B.50d.txt
Glove.6B.zip is being downloaded>>>>

it stuck here

**Comment by Sunil13kush** [comment]
i am using it with python virtual environment not using conda
